.env.example

```
@@ -1,2 +1,2 @@
-EXPO_PUBLIC_SUPABASE_URL=project_url
-EXPO_PUBLIC_SUPABASE_PUBLISHABLE_KEY=project_publishable_key. 
\ No newline at end of file
+EXPO_PUBLIC_SUPABASE_URL=https://cvofzbwyndvpnaeeeahx.supabase.co
+EXPO_PUBLIC_SUPABASE_PUBLISHABLE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImN2b2Z6Ynd5bmR2cG5hZWVlYWh4Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTYyNjEwODgsImV4cCI6MjA3MTgzNzA4OH0.dbNiaLi90DDZiTSV4YjYBoUN6Vc9Yz5pBAXkT7pXzd0
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)